yb-voyager/cmd/live_migration.go

```diff
@@ -308,10 +308,12 @@ func processEvents(chanNo int, evChan chan *tgtdb.Event, lastAppliedVsn int64, d
 				}
 				if event.Vsn <= lastAppliedVsn {
 					log.Tracef("ignoring event %v because event vsn <= %v", event, lastAppliedVsn)
+					conflictDetectionCache.RemoveEvents([]*tgtdb.Event{event})
```

Comment:
We should change the function to a variadic function here:
```
func (cfdc *ConflictDetectionCache) RemoveEvents(events ...*tgtdb.Event) {

}
```

and here simply pass it as:
```
conflictDetectionCache.RemoveEvents(event)
```
and
```
conflictDetectionCache.RemoveEvents(eventBatch.Events...)
```